packages/beacon-node/src/execution/engine/mock.ts

```diff
@@ -404,6 +406,12 @@ export class ExecutionEngineMockBackend implements JsonRpcBackend {
     return versionedHashes.map((_vh) => null);
   }
 
+  private getBlobsV2(
+    _versionedHashes: EngineApiRpcParamTypes["engine_getBlobsV2"][0]
+  ): EngineApiRpcReturnTypes["engine_getBlobsV2"] {
+    return [];
```

Comment:
our mock doesn't seem to follow the spec as we should return null if blobs are missing, or why is this an empty array, ie. if the block has 0 blobs (no commitments) then we shouldn't even query the blobs api on the EL

```suggestion
    return null;
```

Reply:
updated in #8164